css/style.css

```diff
@@ -100,6 +97,252 @@ ul li a {
   }
 
   .small {
-    display: block;
+    display: inline-block;
+    padding-right: 30px;
+  }
+}
+
+.work {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.fa-angle-down {
+  font-size: 25px;
+  padding: 5px 5px;
+  border: 1px solid rgba(0, 0, 0, 0.63);
+  border-radius: 50% 50%;
+}
+
+.work-title {
+  font-family: 'Inter', sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: center;
+}
+
+.work-title::after {
+  content: "";
+  display: block;
+  width: 48px;
+  height: 4px;
+  background: #36b37e;
+  margin: 15px 45%;
+}
+
+/* cards layout (grid css layout) */
+@media only screen and (max-width: 600px) {
+  .card-layout {
+    display: grid;
+    gap: 1.2em;
+    grid-template-columns: 1fr;
+    justify-content: center;
+  }
+}
+
+@media only screen and (min-width: 600px) {
+  .card-layout {
+    display: grid;
+    gap: 1.2em;
+    grid-template-columns: 1fr 1fr 1fr;
+    justify-content: center;
+  }
+
+  body {
+    background-image: none;
+  }
+}
+
+/* cards (used flexbox to align item) */
+.card {
+  background-color: #ebf0ee;
+  height: 450px;
+  width: 310px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+/* card body */
+.card-body {
+  background-color: #fff;
+  bottom: 0;
+  height: 222px;
+  width: 310px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  border-radius: 8px;
+  padding: 0;
+  margin: 0;
+  gap: 6px;
+}
+
+.card-title {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  padding: 0;
+  margin: 0;
+}
+
+/* card button */
+.see-more {
+  background-color: #36b37f;
+  color: #fff;
+  border-radius: 4px;
+  padding: 15px 10px;
+  bottom: 8px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  margin: 5px 1px;
+}
+
+/* button action hover */
+.see-more:hover {
+  background-color: #65e4af;
+}
+
+/* @media(hover: hover) and (pointer: fine) {
+  .see-more:hover {
+    background-color: #65e4af;
+  }
+} */
```

Comment:
Some nice code here 😎 but in order to make your `CSS` more readable it would be better to remove the comments if you need some piece of code you can still get it back in git history. 🔖